fetcher/fetcher_test.go

```diff
@@ -189,7 +189,7 @@ func TestInitializeAsserter(t *testing.T) {
 				WithRetryElapsedTime(5*time.Second),
 			)
 
-			networkIdentifier, networkStatus, err := f.InitializeAsserter(ctx, test.network)
+			networkIdentifier, networkStatus, err := f.InitializeAsserter(ctx, test.network, "")
```

Comment:
cannot believe Golang does not support default value..

Reply:
🤷